tests/framework/db/ConnectionTest.php

```diff
@@ -178,4 +178,18 @@ public function testTransactionShortcutCustom()
         $this->assertEquals(1, $profilesCount, 'profile should be inserted in transaction shortcut');
     }
 
+    public function testNestedTransaction()
```

Comment:
Should add a link to the issue in method phpdoc since it's not clear what it tests w/o reading the issue.